src/net/rpc.rs

```diff
@@ -1618,7 +1619,14 @@ impl ConversationHttp {
                         .send(http, fd);
                 }
             };
-            let scalar_cost = metric.from_cost_and_len(&estimated_cost, estimated_len);
+            let tip = SortitionDB::get_canonical_burn_chain_tip(sortdb.conn()).unwrap();
+            use vm::database::BurnStateDB;
```

Comment:
Please don't do this.  Put all module `use` statements at the beginning of the file, unless it's truly impossible to do so (e.g. there's an unavoidable name conflict).